proto-wine/ftnoir_protocol_wine_dialog.cpp

```diff
@@ -103,6 +124,63 @@ void FTControls::onWinePathComboUpdated(QString selection) {
     }
 }
 
+void FTControls::onRadioButtonsChanged() {
+    if (ui.variant_wine->isChecked()) {
+        // wine settings selected
+
+        // enable wine settings
+        ui.wine_path_combo->setEnabled(true);
+        ui.wineprefix->setEnabled(true);
+        ui.browse_wine_prefix_button->setEnabled(true);
+        if (ui.wine_path_combo->currentText() == "Custom path to Wine executable") {
```

Comment:
This is bad, absolutely don't do that. Use [QComboBox::itemData](https://doc.qt.io/qt-6/qcombobox.html#itemData) (just stick the actual path in there).

Reply:
In the future we might want to append some sort of indicator to the display text as opposed to the literal name stored in the .ini.

Reply:
Changes have been implemented.
Closing this request tho I still gotta do the Wine and Proton test. Will put a message once I completed those.